src/Prometheus/Storage/Redis.php

```diff
@@ -586,7 +593,7 @@ private function collectGauges(bool $sortMetrics = true): array
         sort($keys);
         $gauges = [];
         foreach ($keys as $key) {
-            $raw = $this->redis->hGetAll(str_replace($this->redis->_prefix(''), '', $key));
+            $raw = $this->redis->hGetAll(str_replace($this->prefix(''), '', $key));
```

Comment:
`$this->prefix('')` is always an empty string for this adater, but `$key` contains the prefix to remove. 
perhaps `$this->removePrefixFromKey($key)` was intended here ?
The same applies to collectHistograms, collectGauges
```suggestion
            $raw = $this->redis->hGetAll($this->removePrefixFromKey($key));
```